mentorApp/lib/Navigation/AppController.dart

```diff
@@ -1,8 +1,8 @@
 import 'package:curved_navigation_bar/curved_navigation_bar.dart';
 import 'package:flutter/material.dart';
-import "./Profile/UserProfile.dart";
-import './Dashboard/Dashboard.dart';
-import "./Chat/Chat.dart";
+import "../Profile/UserProfile.dart";
```

Comment:
Why is it `import "../Profile/UserProfile.dart";` instead of `import "./Profile/UserProfile.dart";`?

Reply:
The `..` represents moving back one directory/folder.